api/request.go

```diff
@@ -263,7 +263,7 @@ func (h AttributeRequest) Hash() (string, error) {
 func (h AttributeRequest) toString() (string, error) {
 	msg := "{vds: %s, Rotation: %.2f, Origin: [%.2f, %.2f], " +
 		"Increment: [%.2f, %.2f], FillValue: %.2f interpolation: %s, " +
-		"Above: %f, Below: %f, Stepsize: %f, Attributes: %v}"
+		"Above: %2f, Below: %2f, Stepsize: %2f, Attributes: %v}"
```

Comment:
What's the difference between %2f and %.2f ?
I see only dot in the examples.

Reply:
Should be dot